projects/plugins/boost/app/lib/critical-css/Critical_CSS_State.php

```diff
@@ -160,6 +160,15 @@ public function set_as_failed( $error ) {
 		$this->save();
 	}
 
+	/**
+	 * Get Critical CSS state status.
+	 *
+	 * @return mixed
+	 */
+	public function get_state_status() {
```

Comment:
It's probably just me and let me know if you disagree. Since the class is already `Critical_CSS_State` perhaps we could just call `get_status`. I guess, having a "state" property in a class named "state" made things already pretty confusing. 🤪

Reply:
Oops, you're absolutely right. I was bouncing between classes and totally missed, that there's already state in there haha.